src/utils/update/fetch.beneficial.owners.private.data.ts

```diff
@@ -31,20 +31,20 @@ export const fetchBeneficialOwnersPrivateData = async (appData: ApplicationData,
 };
 
 const mapBeneficialOwnersPrivateData = (boPrivateData, appData: ApplicationData) => {
-  if (boPrivateData === undefined || boPrivateData.length === 0) {
-    return;
-  }
-  const mapUsualResidentialAddress = beneficialOwner => {
+  appData.update?.review_beneficial_owners_individual?.forEach(beneficialOwner => {
     if (beneficialOwner.ch_reference) {
-      beneficialOwner.usual_residential_address = mapBoPrivateAddress(boPrivateData, beneficialOwner.ch_reference, false);
+      mapIndividualBOPrivateData(boPrivateData, beneficialOwner);
     }
-  };
-  const mapPrincipalAddress = beneficialOwner => {
+  });
+  appData.update?.review_beneficial_owners_corporate?.forEach(beneficialOwner => {
     if (beneficialOwner.ch_reference) {
-      beneficialOwner.principal_address = mapBoPrivateAddress(boPrivateData, beneficialOwner.ch_reference, true);
+      mapCorporateOrGovernmentBOPrivateData(boPrivateData, beneficialOwner);
     }
-  };
-  appData.update?.review_beneficial_owners_individual?.forEach(mapUsualResidentialAddress);
-  appData.update?.review_beneficial_owners_government_or_public_authority?.forEach(mapPrincipalAddress);
-  appData.update?.review_beneficial_owners_corporate?.forEach(mapPrincipalAddress);
+  });
+  appData.update?.review_beneficial_owners_government_or_public_authority?.forEach(beneficialOwner => {
```

Comment:
Apologies if I previously asked. Is there a test for `review_beneficial_owners_government_or_public_authority` been undefined? 
Same for the other BO review

Reply:
Added some tests.